pkg/internal/mock_plaid/verification.go

```diff
@@ -0,0 +1,43 @@
+package mock_plaid
+
+import (
+	"encoding/json"
+	"github.com/brianvoe/gofakeit/v6"
+	"github.com/monetr/rest-api/pkg/internal/mock_http_helper"
+	"github.com/monetr/rest-api/pkg/internal/myownsanity"
+	"github.com/plaid/plaid-go/plaid"
+	"github.com/stretchr/testify/require"
+	"net/http"
+	"testing"
+	"time"
+)
+
+func MockGetWebhookVerificationKey(t *testing.T) {
+	mock_http_helper.NewHttpMockJsonResponder(t,
+		"POST", Path(t, "/webhook_verification_key/get"),
+		func(t *testing.T, request *http.Request) (interface{}, int) {
+			ValidatePlaidAuthentication(t, request, DoNotRequireAccessToken)
+			var getWebhookVerificationKeyRequest struct {
+				KeyId string `json:"kid"`
+			}
+			require.NoError(t, json.NewDecoder(request.Body).Decode(&getWebhookVerificationKeyRequest), "must decode request")
+
+			return plaid.WebhookVerificationKeyGetResponse{
+				Key: plaid.JWKPublicKey{
+					Alg:       "",
+					Crv:       "",
+					Kid:       "",
+					Kty:       "",
+					Use:       "",
+					X:         "",
+					Y:         "",
```

Comment:
Definitely need to actually finish this. Or it needs to be removed as it is not valuable.

Reply:
Going to leave this blank for now, but I did correct the request json.

I need to learn how JWKS authentication works properly. I don't want to implement it now and get it wrong.